transcom/mymove
# MB-16343 add fsc to sit service item bundles

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16343)

## Summary
When 1st day origin/destination sit service items are created, 2 other service items are created along side it. This ticket creates a SIT FSC service item along with `Additional Day SIT` and `Pickup/Delivery SIT`

- [ ] Let Team Huckleberry know about the changes
- [ ] Auto-create new SIT FSC services
- [ ] make sure DOSFSC has relevant original/actual addresses saved (and any other fields that are populated when creating the origin sit bundle)
- [ ] make sure DDSFSC has relevant original/actual addresses saved (and any other fields that are populated when creating the dest sit bundle)
- [ ] Update the prime.yaml description for create mto service item POST section
- [ ] Watch out for broken playwright tests
- [ ] Run through manual test plan to verify functionality

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Login as prime simulator
2. Look at `MISHIP` or any other move with approved shipments
3. Click on Add Service Item button
4. Select Origin SIT for service item type field and fill out form
5. Click Create Service Item button
6. Repeat steps 4 and 5 for Destination SIT service item type
7. Navigate back to prime simulator landing page and look at the move again
8. Click on Create Payment Request button
9. Scroll down to see service items for SIT origin/destination FSC


### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
